model_signing/signing/testdata/in_toto/TestSingleDigestIntotoPayload/deep_model_folder

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
nit: maybe add the extension `.intoto` to the file, to give a hint to readers of what the file contains

Reply:
Let me do this after merging all of these, to do it across the board (rebase would be harder otherwise, with a lot of conflicts due to file moves).

Reply:
FYI: If you decide to rename, this would go against the [naming convention](https://github.com/in-toto/attestation/blob/main/spec/v1/envelope.md#file-naming-convention). So maybe `.intoto.json` ?

> If stored in a dedicated file by itself, and not as part of a [Bundle](https://github.com/in-toto/attestation/blob/main/spec/v1/bundle.md), an Envelope SHOULD use the suffix .json.

Reply:
Oh, that's correct. Yes, it should be `.intoto.json`